developer-hub/src/lib.rs

```diff
@@ -0,0 +1,161 @@
+//! # Developer Hub
+//!
+//! The Polkadot SDK Developer Hub.
+//!
+//! This crate is a *minimal*, but *always-accurate* source of information for those wishing to
+//! build on the Polkadot SDK.
+//!
+//! > **Work in Progress**: This crate is under heavy development. Expect content to be moved and
+//! > changed. Do not use links to this crate yet.
+//!
+//! ## Getting Started
+//!
+//! We suggest the following reading sequence:
+//!
+//! - Start by learning about the structure of the [`polkadot_sdk`] and its context.
+//! - Then, head over the [`tutorial`] to get more hand-on practice.
+//! - Whilst reading the tutorial, you might find back-links to [`reference_docs`].
+//! - Finally, <https://paritytech.github.io> is the parent website of this crate that contains the
+//!   list of further tools related to the Polkadot SDK.
+//!
+//! ## Information Architecture
+//!
+//! This section paints a picture over the high-level information architecture of this crate.
+#![doc = simple_mermaid::mermaid!("../../docs/mermaid/IA.mmd")]
+//!
+//! ## Contribution
+//!
+//! The following sections cover more detailed information about this crate and how it should be
+//! maintained.
```

Comment:
I was already nudged into this direction, and happy to take it even further. One question I have though is that what other content should come here? Most of the cool stuff is in `polkadot_sdk`. 

I can foresee adding more diagrams here. Also a trophy section, naming all the projects that use our code. But arguably all of that can be in `polkadot_sdk`.

All in all, one reason why this `Contributing` section is here is because nothing else fitted :) 

I tried to forward everyone as soon as possible to `polkadot_sdk` though.

Reply:
did 30d6ac2625b9e7e58127141e9aa8c347c77de00e nonetheless, but still want to get more ideas of what should be literally on the first page.